### ISSUE TITLE
Docs on `How to Upgrade`

### ISSUE THREAD

**Issue by zachleat** [11ty/eleventy#373]
Surfaced here https://github.com/11ty/eleventy/issues/361#issuecomment-451904647 by @kleinfreund

**Comment by zachleat** [comment]
This repository is now using [lodash style issue management](https://twitter.com/samselikoff/status/991395669016436736) for documentation requests. This means documentation issues will now be closed instead of leaving them open.

View [the documentation queue backlog here](https://github.com/11ty/eleventy/issues?utf8=%E2%9C%93&q=is%3Aissue+sort%3Areactions-%2B1-desc+is%3Aclosed+label%3Adocumentation+label%3Aneeds-votes). Don’t forget to upvote the top comment with 👍!

**Comment by kleinfreund** [comment]
@zachleat I’d much rather have a few short lines in the actual readme document in the repository. That’s just sometimes the page people (including me) end up on. For example, I often navigate to pages by entering a string like `ele` into the address bar and let my browser suggest me the page I most often visit. For `ele`, that’s https://github.com/11ty/eleventy.

So I’d like to see something like this in the readme:

## Installation & update

```sh
npm install -g @11ty/eleventy
```

---

Basically, the starting lines of the [Getting Started](https://www.11ty.io/docs/getting-started/) document.
